assets/scripts/main.js

```diff
@@ -132,7 +132,7 @@
 
   // Fixes missing anchor on elementor widget pagination.
   // https://github.com/elementor/elementor/issues/4703
-  $(document).ready(() => {
+  document.addEventListener('DOMContentLoaded', () => {
```

Comment:
What this change really means is that this block of the JS should be moved into a separate file and that separate file should be loaded with a higher priority on the page, so that it is loaded before the JS file of Elementor.

In addition, WordPress supports dependencies between scripts. Using them would be even more appropriate.  However, this is a reversed dependency and that is not supported easily. Therefore, loading it with a higher priority is the simplest solution.

Reply:
Can I change it back to jquery and then make the priority 999 for the action?
`add_action('wp_enqueue_scripts', __CLASS__ . '::wp_enqueue_scripts', 999);`